lib/src/r_tree/leaf_node.dart

```diff
@@ -18,6 +18,7 @@ part of r_tree;
 
```

Comment:
I think a solid QA for this would be to see the next PR where all the exported types have been unexported.

Reply:
https://github.com/Workiva/r_tree/pull/65